qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/helper/InitialSetupPage.java

```diff
@@ -152,7 +152,8 @@ public void waitForSetupToComplete() {
 		new WebDriverWait(driver, Duration.ofSeconds(720L))
```

Comment:
```suggestion
		new WebDriverWait(driver, Duration.ofSeconds(720L)).until(ExpectedConditions.or(ExpectedConditions.urlContains("index.htm"),ExpectedConditions.urlContains("home.page")))));
```

Reply:
By having them on one line it exceeds the recommended line width as per the wiki at https://wiki.openmrs.org/display/docs/Developer+How-To+Setup+And+Use+Eclipse